manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/processor/PluginProcessor.java

```diff
@@ -96,6 +106,13 @@
                 WorkflowLogger.endActivity(Status.FAILED);
             }
         });
+
+        Map<ManifestMeta,String> manifestMetaVsSnippets = customSnippetsProvider.fetchUnmergedCustomSnippets();
+        if(manifestMetaVsSnippets != null && !manifestMetaVsSnippets.isEmpty()){
+            manifestMetaVsSnippets.forEach((manifestMeta, snippet)->{
+                WorkflowLogger.warn(ManifestGeneratorActivity.IGNORING_CUSTOM_SNIPPET,manifestMeta.getKind());
```

Comment:
Instead of printing separate message for every resource kind not supported, we can print a single message with all the kinds